packages/amplify-graphql-types-generator/CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Apollo Contributor Guide
```

Comment:
Should remove this file? Looks like apollo contributor guide. We have a contributing block in our main readme, so we can get rid of this.